backend/app/model/user.py

```diff
@@ -26,6 +27,10 @@ class User(db.Model):
     email = db.Column(db.String, nullable=False, unique=True)
     role = db.Column(db.Enum(Role))
     participants = db.relationship(Participant, back_populates="user")
+    participant_count = column_property(select([func.count(Participant.id)]).
+                                        where(Participant.user_id==id).
+                                        correlate_except(Participant)
```

Comment:
I had to look this up, very efficient!

Reply:
I was so excited to learn about it. Now I want to use it everywhere!